CONFIGURATION.md

```diff
@@ -0,0 +1,184 @@
+| `wave.registries.quay.io.password`           | Quay.io password or PAT for authentication.  | `${QUAY_PAT}` | true     |
+| `wave.registries.195996028523.dkr.ecr.eu-west-1.amazonaws.com.username` | AWS ECR username for authentication. | `${AWS_ACCESS_KEY_ID}` | true     |
+| `wave.registries.195996028523.dkr.ecr.eu-west-1.amazonaws.com.password` | AWS ECR password for authentication. | `${AWS_SECRET_ACCESS_KEY}` | true     |
+| `wave.registries.seqeralabs.azurecr.io.username` | Azure Container Registry username for authentication. | `${AZURECR_USER}` | true     |
+| `wave.registries.seqeralabs.azurecr.io.password` | Azure Container Registry password or PAT for authentication. | `${AZURECR_PAT}` | true     |
+
+
+## HTTP client configuration
+
+| Variable                      | Description                                      | Default Value / Environment Variable | Optional |
+|-------------------------------|--------------------------------------------------|---------------|----------|
+| `wave.httpclient.connectTimeout` | Connection timeout for http client.              | `20s`         | false    |
+| `wave.httpclient.retry.delay` | Delay for http client retries.                   | `1s`          | false    |
+| `wave.httpclient.retry.attempts` | Number of http client retry attempts.            | `5`           | false    |
+| `wave.httpclient.retry.maxDelay` | Maximum delay for http client retries.           |               | true     |
+| `wave.httpclient.retry.jitter` | Jitter fot http client retries.                  | `0.25`        | false    |
+| `wave.httpclient.retry.multiplier` | Multiplier for http client retries.              | `1.0`         | false    |
+| `micronaut.http.services.stream-client.read-timeout` | Read timeout for the streaming http client.      | `30s`                                | false    |
+| `micronaut.http.services.stream-client.read-idle-timeout` | Read idle timeout for the streaming http client. | `120s`                           | false    |
+
+
+## Container build process configuration
+
+| Variable                      | Description                                                                                             | Default Value / Environment Variable                 | Optional |
+|-------------------------------|---------------------------------------------------------------------------------------------------------|-----------------------------------------------------|----------|
+| `wave.build.timeout`                  | Timeout for the build process.                                                                          | `5m`                                                | false    |
+| `wave.build.workspace`                | Path to workspace for the build process. e.g. `/efs/wave/build`                                         |                                                     | false    |
```

Comment:
```suggestion
| `wave.build.workspace`                | Path to directory where wave will store build files, e.g. `/efs/wave/build`                                         |                                                     | false    |
```

Not sure this is the correct meaning though, could you review it and expand it?

Reply:
added more description